src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DriverPanelControllerAdapter.java

```diff
@@ -9,12 +9,10 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class DrawPanelControllerAdapter implements Job2dDriver {
+public class DriverPanelControllerAdapter implements Job2dDriver {
```

Comment:
DrawPanelDriver would be better. The goal of the driver was to draw (this should remain as a part of the name)